spec/features/standard_user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+feature 'Standard user collaboraration' do
+  before do
+    premium_user = create(:user, role: "premium")
+    premium_user.update(role: "premium")
```

Comment:
To avoid the need for setting the role twice, `User#default_to_standard` can be modified to:

```ruby
  def default_to_standard
    self.role = "standard" if self.role.blank?
  end 
```